src/providers/twitch/api/Helix.hpp

```diff
@@ -340,6 +340,19 @@ enum class HelixDeleteChatMessagesError {
     Forwarded,
 };
 
+enum class HelixAddChannelModeratorError {
+    Unknown,
+    UserMissingScope,
+    UserNotAuthenticated,
+    UserNotAuthorized,
+    UserIsVIP,
+    Ratelimited,
+    AlreadyModded,
```

Comment:
Whatever prefix you choose for `UserIsVIP` - use it here too (e.g. `TargetAlreadyModded`)